src/libsyntax/ext/tt/macro_parser.rs

```diff
@@ -330,7 +330,8 @@ fn nameize<I: Iterator<Item = NamedMatch>>(
                     }
                 }
             }
-            TokenTree::MetaVar(..) | TokenTree::Token(..) => (),
+            TokenTree::MetaVar(..) |
+            TokenTree::Token(..) => (),
```

Comment:
Nit/meta: it's harder to see important changes when formatting changes are mixed in (quite a few of them in this PR!).

Reply:
This is for consistency, since the code was previously not consistent!